app/controllers/admin/debate_outcomes_controller.rb

```diff
@@ -34,7 +34,8 @@ def fetch_debate_outcome
 
   def debate_outcome_params
     params.require(:debate_outcome).permit(
-      :debated_on, :overview, :transcript_url, :video_url, :debated
+      :debated_on, :overview, :transcript_url, :video_url, :debated,
+      :commons_image
```

Comment:
Fixed. We don't really stick to 80 column width elsewhere, so I've made this the same.